emails/templates/emails/wrapped_email.html

```diff
@@ -123,20 +123,18 @@
           <img width="30" src="{{ SITE_ORIGIN }}/static/images/email-images/relay-icon.png" style="margin-right: 5px; display: inline-block; vertical-align: top;" alt="relay icon"/>   
            
           <p style="margin-top: 0; margin-bottom: 5px; display: inline-block;"> 
-            {% with display_email|striptags|urlencode as mask_url %}
             <span class="forwarded-from-email" style="display: block; color: #FFFFFF;"> 
-            {% ftlmsg 'relay-email-forwarded-from' %} 
-            <a class="container-link" href="{{ SITE_ORIGIN }}/accounts/profile/#{{ mask_url }}" style="margin-right: 30px;color: #FFFFFF;"> {{ display_email|striptags }} </a>
+            {% with display_email|striptags|urlencode as mask_url %}
+            {% ftlmsg 'relay-email-forwarded-from-html' url=SITE_ORIGIN|add:'/accounts/profile/#'|add:mask_url attrs='class="container-link" style="margin-right: 30px;color: #FFFFFF;"' email_address=display_email|striptags %}
```

Comment:
change (blocking): Don't change the template yet until the message has been translated into our "top supported languages". I'm not sure what those are for email-receiving users, but our top 10 languages for web traffic are:

<img width="86" alt="image" src="https://github.com/mozilla/fx-private-relay/assets/71928/7928d504-571b-4cf2-8849-c7d7861f0738">

So maybe German, French, Russian, Spanish, Portuguese (Brazil), and Italian? Those cover 82% of our traffic.

If we change the template from an existing ftl msg ID that is translated to one that is only in the `pending.ftl` file, this will effectively "un-translate" this message in the email wrapper until translators have translated this new message, and we release the new translations.

I think the best practice here is to add this new ftl msg to the file for translation, and then after it's been translated into enough langauges, change the template.

Reply:
None of the `relay-email-` strings have been submitted to the translation repository, so there are no existing translations that will break